Norwegian

aslak47 opened the pull request:
I have added soft hyphens to long words within the section [Text] of file lang/no.conf.
I removed some other lines that are not used, and I made a couple of other corrections as well.
Trying to add soft hyphens to labels used in LiveGauges and LiveCharts was not successful. The text "&shy;" was shown as part of the labels of gauges and charts.

mKainzbauer commented:
Yes, the hyphens are kind of problemativ for charts and gauges. I'll have to take a look how to get along with this.

mKainzbauer commented:
I decoded the text for the charts, now it should work.

mKainzbauer commented:
For the double quots ("): you should need them in most cases only in the Texts. So leave them out, except for the texts, refer to lang/de.conf

aslak47 commented:
Thank you again.
I now added soft hyphens to all long words, and I corrected &hsy; to &shy; in initial comment.
In addition I removed all generated html files, restarted the station and cleaned the buffer of web browser. Now it works fine.

mKainzbauer commented:
Hi @aslak47 
There is another issue with the hyphens: https://github.com/brewster76/fuzzy-archer/issues/95 Well.... :) I need to thinks about that... as for now, remove the soft hyphen on your personal configs, if they interfere.
